content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -494,31 +494,35 @@ This process (steps 3-6) can also be triggered with `kubeadm phase selfhosting c
 
 For running kubeadm without an internet connection you have to pre-pull the required master images for the version of choice:
 
-| Image Name                                               | v1.8 release branch version | v1.9 release branch version |
-|----------------------------------------------------------|-----------------------------|-----------------------------|
-| k8s.gcr.io/kube-apiserver-${ARCH}          | v1.8.x                      | v1.9.x                      |
-| k8s.gcr.io/kube-controller-manager-${ARCH} | v1.8.x                      | v1.9.x                      |
-| k8s.gcr.io/kube-scheduler-${ARCH}          | v1.8.x                      | v1.9.x                      |
-| k8s.gcr.io/kube-proxy-${ARCH}              | v1.8.x                      | v1.9.x                      |
-| k8s.gcr.io/etcd-${ARCH}                    | 3.0.17                      | 3.1.10                      |
-| k8s.gcr.io/pause-${ARCH}                   | 3.0                         | 3.0                         |
-| k8s.gcr.io/k8s-dns-sidecar-${ARCH}         | 1.14.5                      | 1.14.7                      |
-| k8s.gcr.io/k8s-dns-kube-dns-${ARCH}        | 1.14.5                      | 1.14.7                      |
-| k8s.gcr.io/k8s-dns-dnsmasq-nanny-${ARCH}   | 1.14.5                      | 1.14.7                      |
-
-Here `v1.8.x` means the "latest patch release of the v1.8 branch".
+| Image Name                                 | v1.9 release branch version | v1.10 release branch version |
+|--------------------------------------------|-----------------------------|------------------------------|
+| k8s.gcr.io/kube-apiserver-${ARCH}          | v1.9.x                      | v1.10.x                      |
+| k8s.gcr.io/kube-controller-manager-${ARCH} | v1.9.x                      | v1.10.x                      |
+| k8s.gcr.io/kube-scheduler-${ARCH}          | v1.9.x                      | v1.10.x                      |
+| k8s.gcr.io/kube-proxy-${ARCH}              | v1.9.x                      | v1.10.x                      |
+| k8s.gcr.io/etcd-${ARCH}                    | 3.1.10                      | 3.1.12                       |
+| k8s.gcr.io/pause-${ARCH}                   | 3.0                         | 3.1                          |
+| k8s.gcr.io/k8s-dns-sidecar-${ARCH}         | 1.14.7                      | 1.14.8                       |
+| k8s.gcr.io/k8s-dns-kube-dns-${ARCH}        | 1.14.7                      | 1.14.8                       |
+| k8s.gcr.io/k8s-dns-dnsmasq-nanny-${ARCH}   | 1.14.7                      | 1.14.8                       |
+| coredns/coredns                            | 1.0.2                       | 1.0.6                        |
```

Comment:
for 1.12 shouldn't we do (1.10, 1.11, 1.12) - i.e. list images for the current one and tracking two versions back?

i wish there was another way to do this...
we might want to eventually auto-generate this doc as an imported `.md` for this page on each release

/cc @xiangpengzhao @dixudx

Reply:
My presumption is, that we keep this table only for versions, that do not support `kubeadm config images list`.

In fact, I would have been more than happy with removing this table altogether.

Reply:
> for 1.12 shouldn't we do (1.10, 1.11, 1.12) - i.e. list images for the current one and tracking two versions back?

Agree to this as per kubeadm support policy.

> My presumption is, that we keep this table only for versions, that do not support kubeadm config images list.
In fact, I would have been more than happy with removing this table altogether.

So we don't need this table from 1.13 cycle :) But maybe we need to optimize the output of `kubeadm config images list`  then?